src/graph/tuning.cc

```diff
@@ -421,11 +421,12 @@ static const double perChMaxTreeBws[][3] = {
   /* Blackwell (N1/N2/N4) */ {2*38.7, 2*41.4, 2*36.0},
 };
 
-NCCL_PARAM(PatEnable, "PAT_ENABLE", 2);
+NCCL_PARAM(PatEnable, "PAT_ENABLE", 0);
 static int ncclPatEnable(struct ncclComm* comm) {
-  int patEnable = ncclParamPatEnable();
+  if (!ncclParamPatEnable()) return 0;
```

Comment:
In the original way, the user could either explicitly disable by setting to 0, explicitly enable by setting to 1, or allow it to selectively disable based on rank count and `netDeviceType` by setting it to 2. We should maintain this behaviour.

```suggestion
  int patEnable = ncclParamPatEnable();
```

Reply:
The reason we removed the original force enable logic here is because with `NCCL_PAT_ENABLE=1` it will still use `RING` algo when > 1ppn, and if we force `NCCL_ALGO=PAT` with it, we get `no algo/proto available for function` error. Since it defaults to ring and PAT is currently only supported for 1ppn, it is better to remove this.

Reply:
How does this work in NCCL?

Reply:
This force enable flag is only to experiment with > 1ppn I assume. We can bring back this when tune RCCL to pick up PAT algo when > 1ppn.

Reply:
Right now, there is no use case for this force enable flag and current implementation should be fine.